src/resources/help/en_US/relnotes3.0.3.html

```diff
@@ -27,6 +27,7 @@
   <li>Fixed bug with VTK reader parsing .vtm files when 'DataSet' tag doesn't have a 'file' attribute.</li>
   <li>Disable VCR play/reverse play buttons when there is not active drawn plot.</li>
   <li>Fixed inability to Pick on glyphed points lying near the dataset bounds.</li>
+  <li>Fixed the issue with installing host profiles after deleting or moving the .visit folder while VisIt is running.</li>
```

Comment:
Is the "while VisIt is running" necessary.

Reply:
I just realized that VisIt probably creates the ".visit" directory when it runs, so the only way it can be missing is if it deleted or moved while VisIt is running, so the clause is appropriate.

Reply:
Yes...this issue only happened when VisIt was running since the parent directories need to be recreated which mkdir didn't do. If the .visit dir is moved when VisIt is not running, during startup VisIt would create the parent directories.